website/static/js/accountSettings.js

```diff
@@ -193,6 +209,21 @@ var UserProfileViewModel = oop.extend(ChangeMessageMixin, {
             this.changeMessage('Email cannot be empty.', 'text-danger');
         }
     },
+    reSendConfirmation: function(email){
+        var self = this;
+        self.changeMessage('', 'text-info');
+        bootbox.confirm({
+            title: 'Resend Email Confirmation?',
+            message: 'Are you sure that you want to resend email confirmation at ' + '<em><b>' + email.address() + '</b></em>',
+            callback: function (confirmed) {
+                if (confirmed) {
+                    self.client.update(self.profile(), email).done(function () {
+                        $osf.growl('Email confirmation resent to ', '<em>' + email.address() + '<em>', 'success');
```

Comment:
The growlbox looks awkward:

![osf___account_settings](https://cloud.githubusercontent.com/assets/2379650/8109197/7e88f296-1023-11e5-9d5d-066631230ce4.png)

It should be made consistent with the other growlbox for sending a confirmation link:

![osf___account_settings](https://cloud.githubusercontent.com/assets/2379650/8109229/993a6ad4-1023-11e5-915c-4a1679f8789b.png)